apps/agent/contracts/Agent.sol

```diff
@@ -35,6 +35,7 @@ contract Agent is IERC165, ERC1271Bytes, IForwarder, IsContract, Vault {
     uint256 public constant PROTECTED_TOKENS_CAP = 10;
 
     bytes4 private constant ERC165_INTERFACE_ID = 0x01ffc9a7;
+    bytes4 private constant ERC721_RECEIVED = 0x150b7a02; // bytes4(keccak256("onERC721Received(address,address,uint256,bytes)"))
```

Comment:
Maybe rename the variable to `ERC721_RECEIVED_INTERFACE_ID` to be consistent with the `ERC165_INTERFACE_ID`.